### PR TITLE
Update ansible with new scheduler instructions

### PR DESCRIPTION
**Pull request by hunhoffe**
Add instructions to add configuration for akka dispatcher, add complete file paths in two places

## Description
When trying to deploy the new scheduler with ansible, I kept getting this error in the scheduler:
```
Exception in thread "main" akka.ConfigurationException: Dispatcher [dispatchers.lease-service-dispatcher] not configured for path akka://scheduler-actor-system/user/$b
```
Adding the dispatcher to the references.conf file fixed the error, and now I can deploy the scheduler via ansible.
I've just tested this locally by invoking a few functions after deployment.

## My changes affect the following components
- [ ] API
- [ ] Controller
- [ ] Message Bus (e.g., Kafka)
- [ ] Loadbalancer
- [ ] Scheduler
- [ ] Invoker
- [ ] Intrinsic actions (e.g., sequences, conductors)
- [ ] Data stores (e.g., CouchDB)
- [ ] Tests
- [ ] Deployment
- [ ] CLI
- [ ] General tooling
- [X] Documentation

## Types of changes
- [X] Bug fix (generally a non-breaking change which closes an issue).
- [ ] Enhancement or new feature (adds new functionality).
- [ ] Breaking change (a bug fix or enhancement which changes existing behavior).

## Checklist:
- [X] I signed an [Apache CLA](https://github.com/apache/openwhisk/blob/master/CONTRIBUTING.md).
- [X] I reviewed the [style guides](https://github.com/apache/openwhisk/blob/master/CONTRIBUTING.md#coding-standards) and followed the recommendations (Travis CI will check :).
- [N/A] I added tests to cover my changes.
- [N/A] My changes require further changes to the documentation.
- [X] I updated the documentation where necessary.

### CODE REVIEW

File: ansible/README.md [hunk 1]
```diff
@@ -156,7 +156,7 @@ It will run one more component called "scheduler" and ETCD.
 #### Configure service providers for the scheduler
 You can update service providers for the scheduler as follows. 
 
-**common/scala/src/main/resources**
+**common/scala/src/main/resources/reference.conf**
```
Comment (style95) [inline ansible/README.md:159]: 👍

File: ansible/README.md [hunk 2]
```diff
@@ -176,10 +176,43 @@ whisk.spi {
...
+.
+.
+.
+```
+
 #### Enable the scheduler
 - Make sure you enable the scheduler by configuring `scheduler_enable`.
 
-**ansible/environments/local/group_vars**
+**ansible/environments/local/group_vars/all**
```
Comment (style95) [inline ansible/README.md:215]: 👍

File: ansible/README.md [hunk 3]
```diff
@@ -176,10 +176,43 @@ whisk.spi {
 .
 ```
 
+#### Configure akka dispatcher for the scheduler
+Add a new dispatcher entry as follows.
+
+**common/scala/src/main/resources/reference.conf**
+```
+  lease-service-dispatcher {
```
[the remark this answers is not in the document]
Comment (hunhoffe) [reply ansible/README.md:184]: Thanks for the info, changed to pinned dispatcher in recent commit! I also tested locally after changing to the pinned dispatcher.

**Review by style95 — APPROVED** [review]
LGTM!
Thanks, Erika for the contribution.

**Comment by bdoyle0182** [comment]
LGTM

**Pull request merged**